lib/metasploit/framework/command/console.rb

```diff
@@ -39,12 +39,56 @@ def spinner
     end
   end
 
+  def start_profiler
```

Comment:
As a note, since this is more generic than just the code in this file I would prefer it to live in a more generic place where it could be started (and possibly stopped) independent of the framework console. Somewhere like `lib/metasploit/framework/profile.rb` or `lib/msf/core/profile.rb` should work.

Reply:
Sounds good, thanks!

I've ripped it out, and ensured that it can be called from msfvenom too :+1: